src/components/vuestic-utilities/index.js

```diff
@@ -0,0 +1,24 @@
+import { colorConfig } from '../vuestic-components/va-color-picker/VuesticTheme'
+
+const createBackgroundHelpers = () => {
+  let result = ''
+  Object.keys(colorConfig).map((name) => {
+    result += `.background--${name.toLowerCase()} { background-color: ${colorConfig[name]}; }`
+  })
+  return result
+}
+
+const createTextColorHelpers = () => {
+  let result = ''
+  Object.keys(colorConfig).map((name) => {
+    result += `.text--${name.toLowerCase()} { color: ${colorConfig[name]}; }`
+  })
+  return result
+}
+
+export default (() => {
```

Comment:
This is not how Vue plugins are normally implemented. Check docs.
https://vuejs.org/v2/guide/plugins.html